fiasco/conftest.py

```diff
@@ -33,6 +33,12 @@
 
 @pytest.fixture(scope='session')
 def ascii_dbase_root(tmpdir_factory):
+    # If we already have a local copy, just return the path to that
+    local_dbase_path = os.path.join(FIASCO_HOME, 'chianti_dbase')
```

Comment:
`fiasco.defaults` stores the settings in the `fiascorc` file so we should probably check there instead of hardcoding here. It will default to `FIASCO_HOME / 'chianit_dbase'` anyway.

Reply:
I'll just merge this and fix this on #86